tests/general/AnnotationBearTest.py

```diff
@@ -0,0 +1,81 @@
+from queue import Queue
+import unittest
+
+from bears.general.AnnotationBear import AnnotationBear
+from coalib.results.SourceRange import SourceRange
+from coalib.results.AbsolutePosition import AbsolutePosition
+from coalib. results.HiddenResult import HiddenResult
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from tests.LocalBearTestHelper import execute_bear
+
+
+class AnnotationBearTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section1 = Section("")
+        self.section1.append(Setting('language', 'python3'))
+        self.section1.append(Setting('language_family', 'python3'))
+        self.uut1 = AnnotationBear(self.section1, Queue())
+        self.section2 = Section("")
+        self.section2.append(Setting('language', 'c'))
+        self.section2.append(Setting('language_family', 'c'))
+        self.uut2 = AnnotationBear(self.section2, Queue())
+
+    def test_single_line_string(self):
+        text = ["'from start till the end with #comments'\n", ]
+        compare = (SourceRange.from_absolute_position(
```

Comment:
yes, same as earlier, maybe make the indendation on the other lines start from ``= (`` ? :D